pkg/codegen/python/gen.go

```diff
@@ -1528,14 +1539,24 @@ func (mod *modContext) genMethods(w io.Writer, res *schema.Resource) {
 		}
 
 		// Now simply call the function with the arguments.
-		var typ, ret string
+		var typ string
 		if retTypeNameQualified != "" {
```

Comment:
No need for a change to this PR, just point out: When `shouldLiftReturn` is true, we could consider prefixing the nested result class with an underscore to make it more of an implementation detail, or even consider emitting it inside the method itself to further hide it. But it doesn't matter much in the grand scheme of things, so I'm inclined to keep it as-is for now.